website/pages/docs/concepts/lease.mdx

```diff
@@ -1,13 +1,13 @@
 ---
 layout: docs
-page_title: 'Lease, Renew, and Revoke'
```

Comment:
if you ask Grammarly, it would say it would have the comma there to be correct

https://share.getcloudapp.com/kpuL5ZXL

Reply:
Uff, looks like I stepped into a hornets nest. :D

Reply:
Welcome to Open Source :D